src/EditorFeatures/Core/InlineRename/AbstractEditorInlineRenameService.cs

```diff
@@ -30,154 +21,14 @@ protected AbstractEditorInlineRenameService(IEnumerable<IRefactorNotifyService>
             _globalOptions = globalOptions;
         }
 
-        protected abstract bool CheckLanguageSpecificIssues(
-            SemanticModel semantic, ISymbol symbol, SyntaxToken triggerToken, [NotNullWhen(true)] out string? langError);
-
         public async Task<IInlineRenameInfo> GetRenameInfoAsync(Document document, int position, CancellationToken cancellationToken)
         {
-            var triggerToken = await GetTriggerTokenAsync(document, position, cancellationToken).ConfigureAwait(false);
```

Comment:
this code moved to new SymbolicRenameInfo type at feature layer.  It was a mostly straight move, with mechanical changes that i will call out.